react-native/react/actions/startup.js

```diff
@@ -8,7 +8,9 @@ import engine from '../engine'
 export function startup () {
   return function (dispatch) {
     // Also call getCurrentStatus if the service goes away/comes back.
-    engine.listenOnConnect('getCurrentStatus', () => dispatch(getCurrentStatus()))
+    engine.listenOnConnect('getCurrentStatus', () => dispatch(getCurrentStatus()).catch(error => {
```

Comment:
due to a timing issue i got this to actually reject() which chrome complains about a promise w/ no catch handler that throws...